tailwind.config.mjs

```diff
@@ -29,8 +29,11 @@ export default {
         ],
       },
       fontSize: {
+        tiny: ['10px'],
```

Comment:
Yep, I know. So many disorganized font sizes. We'll get there! First, describe the system...

Reply:
Would be useful to add a short comment to note that these sizes are describing the current usage rather than prescriptive set from a design system.